tests/contrib/gunicorn/wsgi_mw_app.py

```diff
@@ -1,20 +1,59 @@
+"""
+This app exists to replicate and report on failures and degraded behavior that can arise when using ddtrace with
+gunicorn
+"""
+import json
+import os
+import sys
+
 from ddtrace import tracer
 from ddtrace.contrib.wsgi import DDWSGIMiddleware
+from ddtrace.internal.compat import PY3
+from ddtrace.internal.remoteconfig import RemoteConfig
+from ddtrace.profiling import bootstrap
+import ddtrace.profiling.auto  # noqa
 from tests.webclient import PingFilter
 
 
+if os.getenv("_DD_TEST_IMPORT_SITECUSTOMIZE"):
+    import ddtrace.bootstrap.sitecustomize  # noqa: F401  # isort: skip
+
 tracer.configure(
     settings={
         "FILTERS": [PingFilter()],
     }
 )
 
+if sys.version_info[1] < 11:
+    from ddtrace.debugging import DynamicInstrumentation
+
+
+def aggressive_shutdown():
+    RemoteConfig.disable()
+    if sys.version_info[1] < 11:
+        DynamicInstrumentation.disable()
+    if PY3:
```

Comment:
I don't think we're running these tests with python2. Is this PY3 check required?